packages/discovery-provider/src/tasks/entity_manager/entities/comment_notification_setting.py

```diff
@@ -0,0 +1,25 @@
+from src.models.comments.comment_notification_setting import CommentNotificationSetting
+from src.tasks.entity_manager.utils import Action, EntityType, ManageEntityParameters
+
+
+def update_comment_notification_setting(params: ManageEntityParameters):
```

Comment:
Was there any particular reason to not put this alongside the other comment things in `entities/comment.py`? Seems like it should go in there?

Reply:
yeah going to move it back, was thinking that file was getting large, but this isnt actually helpful